doc/examples/external_simulators.ipynb

```diff
@@ -0,0 +1,238 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Script based external simulators"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook, we demonstrate the usage of the script based external simulators, summary statistics, and distance functions features.\n",
+    "\n",
+    "These allow to use near-arbitrary programing languages and output for pyABC. The main concept is that all communication happens via the file system. This comes at the cost of a considerable overhead, making this feature not optimal for models with a low simulation time. For more expensive models, the overhead should be negligible."
```

Comment:
"This comes at the cost of a considerable overhead, making this feature not optimal for models with a low simulation time. For more expensive models, the overhead should be negligible"

I think it is the other way around, right?
low simulation time -> good
expensive models -> considerable overhead

Reply:
no, it's the way I meant it: if the simulation time is low, then the relative impact of always going via the file system is high, whereas you will hardly notice the overhead when the model simulation already takes minutes.

Reply:
ah! okay I see what you mean :)